app/controllers/api/v1/accounts/custom_filters_controller.rb

```diff
@@ -26,6 +31,14 @@ def destroy
 
   private
 
+  def update_filter_conversation_count(filter)
+    records = filter.filter_records
+
+    Redis::Alfred.set(filter.filter_count_key, records[:count][:all_count])
+
+    Rails.configuration.dispatcher.dispatch(CUSTOM_FILTER_UPDATED, Time.zone.now, custom_filter: filter)
+  end
+
```

Comment:
We already have this logic in `custom_filter.rb` no need to duplicate it here again.